app/Http/RulesValidation/AnnouncementRules.php

```diff
@@ -41,7 +41,8 @@ trait AnnouncementRules
         'announcement_title' => 'required|string',
         'job_description' => 'required|string',
         'property' => 'required|string',
-        'picture' => 'nullable|mimes:jpeg,jpg,png,gif|max:50000',
+        'file_picture' => 'present|mimes:jpeg,jpg,png,gif|max:5242880|nullable',
```

Comment:
```suggestion
        'file_picture' => 'nullable|mimes:jpeg,jpg,png,gif|max:5242880',
```
ฝากเอา present ออกให้หน่อย ไม่จำเป็นต้องมีล่ะ